pkgs/development/python-modules/google-api-python-client/default.nix

```diff
@@ -16,7 +16,11 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     google-auth google-auth-httplib2 google-api-core
-    httplib2 six uritemplate oauth2client
+    httplib2 six uritemplate oauth2client setuptools
+  ];
+
+  pythonImportsCheck = [
+    "setuptools"
```

Comment:
this should be trying to import the package, as there may be other packages missing

Reply:
As in the pythonImportsCheck should be for "pkg_resources" (which is the missing import seen in https://github.com/NixOS/nixpkgs/pull/104919#issuecomment-758441324)? I suppose that would make more sense.

Reply:
No, wait, you're saying it should be "google-api-python-client"?

Reply:
Yes. Totally did not properly read this.